src/docs/reusable-code-and-services/reusable-services-list.md

```diff
@@ -11,204 +11,222 @@ page_purpose: Provides brief details on specific reusable services and links out
 
 audience: developer, technical lead
 
-author: Jonathan Bond
+author: Jonathan Bond, Shelly Han
 
 content_owner: Olena Mitovska
 
 sort_order: 2
 ---
 # Reusable services list
-Find details on the following services or tools you can use as part of your project in `bcgov`.
+Last updated: **November 22, 2023**
+
+This document covers some of the most popular reusable components designed to enhance the feature and infrastructure of your applications. You can find out more about the services and tools below, and guidance on how to integrate them into your application. This list crosses with the [BCGov Common Components](https://digital.gov.bc.ca/common-components/) as well as the [BCGov Software as a Service (SaaS) catalog](https://digital.gov.bc.ca/cloud/services/saas/), feel free to check out from all three places when looking for a component to use!
 
 ## On this page
```

Comment:
@ShellyXueHan do you think pipline-templates could/should be added to this list? https://github.com/bcgov/pipeline-templates